lib/firefox/webdriver/builder.js

```diff
@@ -168,9 +169,36 @@ export async function configureBuilder(builder, baseDir, options) {
   // https://bugzilla.mozilla.org/show_bug.cgi?id=1751196
 
   if (!isAndroidConfigured(options)) {
-    ffOptions.setBinary(
-      firefoxTypes.length > 0 ? firefoxTypes[0] : Channel.RELEASE
-    );
+    // Fix for https://github.com/sitespeedio/browsertime/issues/2041
+    if (platform() === 'darwin') {
+      if (
+        !firefoxConfig.binaryPath &&
+        !firefoxConfig.nightly &&
+        !firefoxConfig.beta &&
+        !firefoxConfig.developer
+      ) {
+        ffOptions.setBinary('/Applications/Firefox.app/Contents/MacOS/firefox');
```

Comment:
@soulgalore actually it would be better if you only set the app bundle on MacOS. This would prevent issues with the real binary that you usually should read out of the `info.plist` file. geckodriver itself will find the binary on its own.

Reply:
I'm gonna remove this fix when it's fixed upstream. It was bold releasing 121 the week before Christmas so I just want to get it to work :)

Reply:
Ah, that's fine then. Thanks!

Reply:
It would be better to let Selenium Manager handle this. It will find the Firefox binary on the host and then match it with the appropriate GeckoDriver version.